digits/utils/dforms.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) 2015, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I like using custom fields like this. Any reason we can't put these classes in `utils/forms`? What does `dforms` mean?